// Import from pysc2 protos the definitions for the message that is required:

syntax = "proto2";

// Converter protocol file from pysc2:
// import "pysc2/pysc2/env/converter/proto/converter.proto";

message ConfigureResponse {
  // Whether the converter was successfully configured.
  required bool success = 1;
}

message RandomNumberResponse {
  // A random number generated by the converter.
  required int32 random_number = 1;
}

message Empty {}

service ExternalConverterService {
    // (Method definitions not shown)
    rpc GetRandomNumber(Empty) returns (RandomNumberResponse) {}


    // rpc ConfigureConverter(ConverterSettings) returns (ConfigureResponse) {}

    // rpc ConvertObservation(Observation) returns (ConvertedObservation) {}

}